extensions/src/ACRE2Shared/_CONSTANTS.h

```diff
@@ -15,7 +15,8 @@
 #undef _DEBUG_DIALOG
 
 
-#define PIPE_TIMEOUT 10000
+//#define PIPE_TIMEOUT 10000
+#define PIPE_TIMEOUT 5000 //Lowered this as a 10 second wait before being moved is very noticable. This is however my opinion/preference, so is not required.
```

Comment:
👍 If we can get rid of the commented old timeout and the comment that would be great.